eclipse-scout-core/src/util/files.js

```diff
@@ -48,7 +48,23 @@ export function fileListToArray(fileList) {
   return files;
 }
 
+export function getErrorMessageMaximumUploadSizeExceeded(session, maximumUploadSize) {
+  if (maximumUploadSize < 1024) {
+    return session.text('ui.FileSizeLimit', maximumUploadSize, 'B');
```

Comment:
I think 'Bytes' would be much clearer than 'B'. 

KB, MB and GB are often used abbreviations, but B not so much.